### PR TITLE
[clusterfuzzlite] Make timeout and OOM reporting optional.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
OOMs will be reported by default. Timeouts wont.
Fixes: https://github.com/google/oss-fuzz/issues/6703
Should also fix: #6619
Fixes: https://github.com/google/oss-fuzz/issues/3432

### CODE REVIEW

**Comment by evverx** [comment]
I'm not sure about OOMs but I think if `REPORT_TIMEOUTS` is set to `false` CIFuzz shouldn't pass `-t 25` to libFuzzer to let it ignore timeouts and keep fuzzing at the same time.

**Comment by jonathanmetzman** [comment]
> I'm not sure about OOMs but I think if `REPORT_TIMEOUTS` is set to `false` CIFuzz shouldn't pass `-t 25` to libFuzzer to let it ignore timeouts and keep fuzzing at the same time.

This is an interesting point. But it will be very annoying to fix this in libClusterFuzz (I don't think I will have time for this soon).

File: infra/cifuzz/fuzz_target.py [hunk 1]
```diff
@@ -254,26 +256,31 @@ def is_reproducible(self, testcase, target_path):
...
 
-  def is_crash_reportable(self, testcase):
+  def is_crash_reportable(self, testcase, reproduce_args):
     """Returns True if a crash is reportable. This means the crash is
     reproducible but not reproducible on a build from the ClusterFuzz deployment
     (meaning the crash was introduced by this PR/commit/code change).
 
     Args:
       testcase: The path to the testcase that triggered the crash.
+      reproduce_args: The arguments to pass to the target to reproduce the
```
Comment (radarhere) [inline infra/cifuzz/fuzz_target.py:266]:
```suggestion
      reproduce_args: The arguments to pass to the target to reproduce the
      crash.
```

Comment (jonathanmetzman) [reply infra/cifuzz/fuzz_target.py:266]: Thanks!

**Comment by jonathanmetzman** [comment]
> > I'm not sure about OOMs but I think if `REPORT_TIMEOUTS` is set to `false` CIFuzz shouldn't pass `-t 25` to libFuzzer to let it ignore timeouts and keep fuzzing at the same time.
> 
> This is an interesting point. But it will be very annoying to fix this in libClusterFuzz (I don't think I will have time for this soon).

I think the least painful way to do this is to pass an extra timeout argument to libClusterFuzz, I'll file a bug and save that for  another PR. Don't think I can fix this issue this week.

**Review by oliverchang — APPROVED** [review]

**Pull request merged**